ansible_collections/arista/avd/molecule/eos_cli_config_gen_negative_unit_tests/inventory/host_vars/hardware-counters.yml

```diff
@@ -5,4 +5,4 @@ hardware_counters:
   features:
     - qos: in
 
-expected_error_message: "The data model 'hardware_counters' changed in AVD4.0.0. 'hardware_counters.features.name' was expected but not set!"
+expected_error_message: "The data model 'hardware_counters' changed in AVD4.0.0. 'hardware_counters.features.name' was expected but not set."
```

Comment:
```suggestion
expected_error_message: "The data model 'hardware_counters' changed in AVD 4.0.0. 'hardware_counters.features.name' was expected but not set."
```

Reply:
This would require also changing the code that generates this error.